code/_globalvars/bitfields.dm

```diff
@@ -85,16 +85,21 @@ DEFINE_BITFIELD(clothing_flags, list(
 	"BLOCKS_SPEECH" = BLOCKS_SPEECH,
 	"BLOCK_GAS_SMOKE_EFFECT" = BLOCK_GAS_SMOKE_EFFECT,
 	"CASTING_CLOTHES" = CASTING_CLOTHES,
+	"DANGEROUS_OBJECT" = DANGEROUS_OBJECT,
 	"GAS_FILTERING" = GAS_FILTERING,
+	"HEADINTERNALS" = HEADINTERNALS,
+	"INEDIBLE_CLOTHING" = INEDIBLE_CLOTHING,
+	"LARGE_WORN_ICON" = LARGE_WORN_ICON,
 	"LAVAPROTECT" = LAVAPROTECT,
 	"MASKINTERNALS" = MASKINTERNALS,
 	"NOSLIP" = NOSLIP,
+	"PLASMAMAN_HELMET_EXEMPT" = PLASMAMAN_HELMET_EXEMPT,
+	"PLASMAMAN_PREVENT_IGNITION" = PLASMAMAN_PREVENT_IGNITION,
 	"SNUG_FIT" = SNUG_FIT,
 	"STOPSPRESSUREDAMAGE" = STOPSPRESSUREDAMAGE,
 	"THICKMATERIAL" = THICKMATERIAL,
 	"VOICEBOX_DISABLED" = VOICEBOX_DISABLED,
 	"VOICEBOX_TOGGLABLE" = VOICEBOX_TOGGLABLE,
-	"INEDIBLE_CLOTHING" = INEDIBLE_CLOTHING,
```

Comment:
why are you removing this?

Reply:
I didn't, I just moved it up in alphabetical order.

Reply:
You passed the memory check, good work